src/main/scala/org/scalameter/Persistor.scala

```diff
@@ -6,15 +6,15 @@ package org.scalameter
 
 
 trait Persistor {
-  def load(context: Context): History
-  def save(context: Context, h: History): Unit
+  def load[T: Manifest](context: Context): History[T]
+  def save[T](context: Context, h: History[T]): Unit
```

Comment:
Btw, along that line - any thoughts about how that plays with the `include` statement?